### ISSUE TITLE
cant save

### ISSUE THREAD

**Issue by saythrt** [Querz/mcaselector#313]
its not a bug i found nowhere to say it , im new , when imma close the application it says , you have unsaved changes , but HOW DO I SAVE CHANGES , no one nowhere says that part , i cant save my changes

**Comment by Matej-Hajek** [comment]
You don't need to save changes, it is done instantly after a change is made.
Please make **backups** of your worlds!!!

**Comment by Morabert** [comment]
> You don't need to save changes, it is done instantly after a change is made. Please make **backups** of your worlds!!!

then why does it pump this message
![Screenshot_175](https://user-images.githubusercontent.com/36409264/166156832-344bcc8e-08df-4b1c-a6e6-b6108d8bd718.png)
?

**Comment by boinkitsbroken** [comment]
Agreed. Such a dialog box should not pop up unless there is a "Save" option in the File menu. Even if the changes are automatically written to file, this is a clear oversight that needs to be corrected.
